lib/core/src/lib/card-view/services/card-view-update.service.ts

```diff
@@ -29,6 +28,10 @@ export interface ClickNotification {
     target: any;
 }
 
+export interface MinimalNode {
```

Comment:
This one seems ok for now, just please don't forget about it later. It seems like a very generic interface, which might be used later in many places. It would be premature to move it to a common place, but if/when the time comes, it is important to not duplicate really the same interfaces (if they are indeed distinct).

Also about the naming... I'm not sure the MinimalNode name is necessary anymore. From the perspective of this component, we are saying: "give men an entity which has an id". Interface name for an object containing id can be something more generic (which is easier and more reasonable to reuse), like:
- `HasId`
- `IdAble`
- etc...

Please feel free to change the interfaces' name to something which is meaningful for their new context. Just because we borrow the already existing interfaces from js-api, their name can be adjusted, most of the times we are only interested in (part of) their structure.

An interface like HasId might be better, because having small interfaces like this let us conform easier to Interface Segregation Pattern, since classes can easily implement multiple interfaces.

Reply:
I will update the interface's name